Resources/ServerInfo/Guidebook/ServerRules/MoffRules/MoffR1.xml

```diff
@@ -0,0 +1,10 @@
+<Document>
+  # Don't ruin other people's fun
+
+  In this server there are less explicit rules on what players can or cannot do.
```

Comment:
```suggestion
  In this server, there are fewer explicit rules on what players can or cannot do.
```

- comma because prepositional clause
- "(less explicit) rules" -> "fewer (explicit rules)" because it's not "the rules are (less explicit)", it's "there are fewer (explicit rules)"